nixos/modules/security/acme.nix

```diff
@@ -193,10 +225,11 @@ in
           servicesLists = mapAttrsToList certToServices cfg.certs;
           certToServices = cert: data:
               let
-                domain = if data.domain != null then data.domain else cert;
+                restartJobs = lib.unique data.systemd.restart;
+                reloadJobs = lib.subtractLists restartJobs (lib.unique data.systemd.reload);
```

Comment:
What is the case where a restart jobs is promoted to reload only? It seems like a configuration error.

Reply:
@zimbatm i 'feel' that there are use-cases but can't think of one atm. so maybe it is over-engineered.

Reply:
many services as `postfix.service` and `dovecot.service` can load new certificates using `reload` so one does not have to cut connections, reset state and such which would be the case with a full reload.

but there could be a plugin for dovecot, a plugin like https://pigeonhole.dovecot.org/ for postifx, which hypotetically would require postfix to restart. i made this up of course, but the code for unique and restart over reload is already there and simple so why not just use it.